js/client-chat.js

```diff
@@ -1427,6 +1427,7 @@
 						// but it's now always applied
 						$messages = this.$chat.find('.chatmessage-' + user);
 						if (!$messages.length) break;
+						if (row[3]) $messages = $messages.slice(-parseInt(row[3], 10));
```

Comment:
```suggestion
						var lineCount = parseInt(row[3], 10) || 0;
						if (lineCount) $messages = $messages.slice(-lineCount);
```

Might as well improve args readability here.